Sources/SwiftSyntaxBuilder/Tokens.swift.gyb

```diff
@@ -33,16 +33,19 @@ enum Tokens {
 
   // MARK: Punctuations and Signs
 
-  /// `":"`
-  static let colon = SyntaxFactory.makeColonToken().withTrailingTrivia(.spaces(1))
-
-  /// `" = "`
-  static let equal = SyntaxFactory.makeEqualToken().withLeadingTrivia(.spaces(1))
-                                                   .withTrailingTrivia(.spaces(1))
+% for token in SYNTAX_TOKENS:
+%   if token.macro_name() == "PUNCTUATOR":
+  /// The `${token.text}` keyword
+  static let `${lowercase_first_word(token.name)}` = SyntaxFactory.make${token.name}Keyword().withTrailingTrivia(.spaces(1))
+%   end
+% end
 
-  /// `"{"`
-  static let leftBrace = SyntaxFactory.makeLeftBraceToken()
+  // MARK: Punctuations and Signs
 
-  /// `"}"`
-  static let rightBrace = SyntaxFactory.makeRightBraceToken()
+% for token in SYNTAX_TOKENS:
+%   if token.macro_name() == "MISC":
```

Comment:
I'm not sure if this is the way to do it? 
Can't find a thing similar to `is_keyword`? Should we add one or should we generate for all tokens and not only `keyword`, `mist` and `punctuator` ?

Reply:
I think I would just differentiate between keywords and non-keywords. I think there’s little benefit in distinguishing punctuators and miscellaneous tokens unless we would have been able to find a general rule regarding whitespace (which we didn’t).